setup.cfg

```diff
@@ -49,6 +50,7 @@ dev =
     pytest-django
     zest.releaser[recommended]
     polydev.github
+    cryptography
```

Comment:
Hm, sure this is a dev requirement ? Shouldn't it be a base requirement ?

Reply:
well spotted, I though it was importer by PyJWT, but is was only an `extras_require`